packages/http/src/webauthn.ts

```diff
@@ -0,0 +1,43 @@
+import type { definitions } from "./__generated__/services/coordinator/public/v1/public_api.types";
+import { get } from "@github/webauthn-json/browser-ponyfill";
+import { base64StringToBase64UrlEncodedString } from "./encoding";
+
+type TWebAuthnStamp = definitions["v1WebAuthnStamp"];
+
+const timeout = 5 * 60 * 1000;
+
+async function getCredentialRequestOptions(
+  payload: string
+): Promise<CredentialRequestOptions> {
+  const challenge = await getChallengeFromPayload(payload);
+
+  return {
+    publicKey: {
+      challenge: await new TextEncoder().encode(challenge),
+      timeout,
+      userVerification: "discouraged",
+    },
+  };
+}
+
+async function getChallengeFromPayload(payload: string): Promise<string> {
+  const messageBuffer = new TextEncoder().encode(payload);
+  const hashBuffer = await crypto.subtle.digest("SHA-256", messageBuffer);
+  const base64String = Buffer.from(hashBuffer).toString("base64");
+  return base64StringToBase64UrlEncodedString(base64String);
+}
+
+export async function getWebAuthnAssertion(payload: string): Promise<string> {
+  const signingOptions = await getCredentialRequestOptions(payload);
+  const clientGetResult = await get(signingOptions);
```

Comment:
nit: let's import this as "webauthnGet"? I had a brief moment of confusion reading this.